ding/utils/render_helper.py

```diff
@@ -0,0 +1,40 @@
+def render(env, render_mode=('rgb_array')):
+    '''
+    Overview:
+        Render the environment's current frame.
+    Arguments:
+        - env (:obj:`BaseEnv`): the ding env.
```

Comment:
Capitalize this sentence, like "- env (:obj:`BaseEnv`): DI-engine env instance."

Reply:
changed